SteamKit2/SteamKit2/Base/Generated/SteamLanguage.cs

```diff
@@ -1826,11 +1827,14 @@ public enum EClientPersonaStateFlag
 		SourceID = 8,
 		Presence = 16,
 		LastSeen = 64,
-		ClanInfo = 128,
+		UserClanRank = 128,
```

Comment:
we should keep the old value with an ObsoleteAttribute for one release. Use `obsolete` instead of `removed`.